lib/samples/apply_class_breaks_renderer_to_sublayer/apply_class_breaks_renderer_to_sublayer_sample.dart

```diff
@@ -28,42 +28,97 @@ class ApplyClassBreaksRendererToSublayerSample extends StatefulWidget {
 class _ApplyClassBreaksRendererToSublayerSampleState
     extends State<ApplyClassBreaksRendererToSublayerSample> {
   final _mapViewController = ArcGISMapView.createController();
+  // create a map with a basemap
+  final map = ArcGISMap.withBasemapStyle(BasemapStyle.arcGISStreets);
+  // set the rendered state to false
+  bool _rendered = false;
+  // set the ready state to false
+  bool _ready = false;
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      body: ArcGISMapView(
-        controllerProvider: () => _mapViewController,
-        onMapViewReady: onMapViewReady,
+      body: SafeArea(
+        child: Column(
+          // add the map view and buttons to a column.
+          children: [
+            Expanded(
+              // add a map view to the widget tree and set a controller.
+              child: ArcGISMapView(
+                controllerProvider: () => _mapViewController,
+                onMapViewReady: onMapViewReady,
+              ),
+            ),
+            SizedBox(
+              height: 60,
+              // add the buttons to the column.
+              child: Row(
+                crossAxisAlignment: CrossAxisAlignment.center,
+                mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                children: [
+                  // apply renderer button
+                  ElevatedButton(
+                    onPressed: !_rendered && _ready ? renderLayer : null,
+                    child: const Text('Change Sublayer Renderer'),
+                  ),
+                  // reset button
+                  ElevatedButton(
+                    onPressed: _rendered ? resetRoutes : null,
+                    child: const Text('Reset'),
+                  ),
+                ],
+              ),
+            )
+          ],
+        ),
       ),
     );
   }
 
-  void onMapViewReady() async {
-    final map = ArcGISMap.withBasemapStyle(BasemapStyle.arcGISStreets);
-
+  void renderLayer() async {
+    // create an image layer
```

Comment:
We are missing the first step in the sample. So the order should be:

1. Create a map and set the sublayer to the operational layers. This will then display the sublayer with it's default renderer (which is a black outline style).
2. Capture the default renderer (`defaultRenderer = subLayer.renderer`) so it can reapplied when the user resets.
3. When the user taps the "change sublayer renderer" button - the renderer is changed to the class breaks renderer (the blue style).
4. When the user taps "reset", the renderer is set back to the default renderer. (`subLayer.renderer = defaultRenderer`).

So when the sample first loads, the layer should be visible and all that's happening is the renderer is being updated / reset.